client/src/ui/BitbakeCommands.ts

```diff
@@ -560,6 +561,33 @@ export async function generateDependencyGraph (bitbakeWorkspace: BitbakeWorkspac
   }
 }
 
+export let isTaskexpStarted = false
+
+export async function examineDependenciesTaskexp (bitbakeWorkspace: BitbakeWorkspace, bitBakeProjectScanner: BitBakeProjectScanner, uri?: any): Promise<void> {
+  if (isTaskexpStarted) {
+    void vscode.window.showInformationMessage('taskexp is already started')
+    return
+  }
+  const chosenRecipe = await selectRecipe(bitbakeWorkspace, bitBakeProjectScanner, uri)
+  if (chosenRecipe !== undefined) {
+    const bitbakeCommand = bitBakeProjectScanner.bitbakeDriver.composeBitbakeCommand(
+      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+      {
+        specialCommand: `bitbake -g ${chosenRecipe} -u taskexp`
+      } as BitbakeTaskDefinition
+    )
+    const command = `nohup bash -c "${bitbakeCommand}"`
```

Comment:
contrarily to toaster, you are starting a foreground process rather than a background one. There is no need for this. Just spawn a regular terminal.

Reply:
Your command can inspire itself from the "Run task for recipe" command. For instance, "menuconfig" on linux-yocto shows how a simple terminal can manage a GUI bitbake command. Try to keep it as simple as that.

Reply:
Done.

Guess I should have questioned further why it did not worked the first time I tried :/